src/3_PB/VerbaliEsterni/VerbaleEsterno_240223/content.typ

```diff
@@ -20,7 +20,13 @@
 ]
 
 = Sintesi Elaborazione Incontro
+Durante l'incontro il gruppo ha chiarito con La Proponente alcuni dubbi che vengono esposti in seguito e ha condiviso con essa lo stato di avanzamento del progetto.
 
-/*************************************/
-/*    INSERIRE SOTTO IL CONTENUTO    */ 
-/*************************************/
+== Chiarimento SAD
+La Proponente, dopo relativa domanda da parte del gruppo, ha chiarito che il documento SAD (Software Architecture Document) richiesto all'interno del capitolato coincide con il documento che il team chiama _Specifiche Tecniche_ e che sta sviluppando in questo momento.
```

Comment:
```suggestion
La Proponente, dopo la relativa interrogazione da parte del gruppo, ha chiarito che il contenuto del documento SAD (Software Architecture Document) richiesto all'interno del capitolato coincide parzialmente con quanto, nel contesto progettuale, il team chiama _Specifiche Tecniche_ e di cui sta effettuando la stesura in questo momento.
```

Siamo noi a chiamarlo o effettiamente si chiama così? in tal caso procederei a renderlo esplicito

Reply:
Altri gruppi lo chiamano Specifiche architetturali o Specifica Tecnica

Reply:
Perché dici che coincide parzialmente?

Reply:
di certo ci sono della cose che ci sono nelle st e non dall'altra parte, tipo tracciamento dei requisiti, 
poi non credo dovremmo fornire loro tale documento. O almeno credo.


Per la questione del nome lo so,  credo specifiche tecniche sia giusto in realtà ma non mi ricordo perchè.

Reply:
Questione del nome ok, ho tolto. Per il resto, col parzialmente mi sembrava poco chiaro il punto, ho modificato la frase, fammi sapere cosa ne pensi.

Reply:
@simocare siamo sicuri che sia stato affermato apertamente? ricordiamoci che poi dorigo deve firmare.
In realtà potrebbe essere un bene che cosi ci leviamo di fare quella parte li

Reply:
@Dovid308 Ho appena riascoltato la registrazione e lo dice, quindi non penso che avrà problemi ad approvare questo verbale.